client/models/machine_configuration.go

```diff
@@ -30,9 +30,6 @@ import (
 // swagger:model MachineConfiguration
 type MachineConfiguration struct {
 
-	// cpu template
```

Comment:
I think we still want to keep the `CPUTemplate` as an option here to allows those that do wish to specify this value to do so. same goes for `Validate` and `validateCPUTemplate` functions